archive/wg-component-standard/component-config/README.md

```diff
@@ -0,0 +1,977 @@
+This copy of [Versioned Component Configuration Files](https://docs.google.com/document/d/1FdaEJUEh091qf5B98HM6_8MS764iXrxxigNIdwHYW9c/)
```

Comment:
Consider using the self-link

```suggestion
This copy of [Versioned Component Configuration Files](https://goo.gl/GM8KyH)
```

or

```suggestion
This copy of [Versioned Component Configuration Files][vccf-proposal]
...

[vccf-proposal]: https://goo.gl/GM8KyH
```

if you want to shorten up this line.

Reply:
Seems like that short URL still works, but there's no way to manage it. Let's keep the full link since it's probably more durable.